opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -1115,7 +1126,7 @@ class TracerProvider(trace_api.TracerProvider):
 
     def __init__(
         self,
-        sampler: sampling.Sampler = _TRACE_SAMPLER,
```

Comment:
Can `_TRACE_SAMPLER` be removed?

Reply:
No. This still needs to remain for the defaulting behavior with regard to the built in samplers.

Reply:
We could rename it though. Was considering moving the defaulting method to this file, too.

Reply:
Already fixed.